ext/bg/js/database.js

```diff
@@ -448,11 +478,7 @@ class Database {
             prefixWildcardsSupported
         };
 
-        {
-            const transaction = db.transaction(['dictionaries'], 'readwrite');
-            const objectStore = transaction.objectStore('dictionaries');
-            await Database._bulkAdd(objectStore, [summary], 0, 1);
-        }
+        await this.bulkAdd('dictionaries', [summary], 0, 1);
```

Comment:
Just a comment about the previous code, that's an interesting way to prevent temporary constants from polluting the entire function scope. As done in this change, another solution was to move the repeated code to a dedicated function.

Reply:
Yeah, I don't even think it was technically necessary in this case.

Reply:
I tested and there was a linter error about redeclaring it below in another block. JS wouldn't need it, though.